hedera-node/src/test/java/com/hedera/services/ServicesAppTest.java

```diff
@@ -114,9 +117,11 @@ void setUp() {
 		given(runningHash.getHash()).willReturn(hash);
 		given(platform.getCryptography()).willReturn(cryptography);
 		given(platform.getSelfId()).willReturn(selfNodeId);
-		given(overridingProps.containsProperty(any())).willReturn(false);
-		given(overridingProps.containsProperty(logDir)).willReturn(true);
-		given(overridingProps.getProperty(logDir)).willReturn("data/recordStreams");
+		if (!nodeProps.containsProperty(logDirKey) || !nodeProps.getStringProperty(logDirKey).matches(logDirVal)) {
```

Comment:
I think we don't need the second condition `!nodeProps.getStringProperty(logDirKey).matches(logDirVal)`